docs/configure.md

```diff
@@ -0,0 +1,104 @@
+| functionName                                        |     Optional      |              | ServerlessCloudFunction Name                                         |
+| serviceName                                         |     Optional      |              | API-Gateway service name, default to create a new serivce            |
+| serviceId                                           |     Optional      |              | API-Gateway service id, if it has will use this APII-Gateway service |
+| code                                                |     Optional      |              | Default is current working directory                                 |
+| [functionConf](#funtionConf-param-description)      |     Optional      |              | Function configure                                                   |
+| [apigatewayConf](#apigatewayConf-param-description) |     Optional      |              | API-Gateway configure                                                |
+
+### funtionConf param description
+
+| Param       | Required/Optional | Default | Description                                                                                                                                     |
+| ----------- | :---------------: | :-----: | :---------------------------------------------------------------------------------------------------------------------------------------------- |
+| timeout     |     Optional      |   3s    | The duration a function allowed to execute. Choose a value between 1 and 300 seconds; The default is 3 seconds.                                 |
+| memorySize  |     Optional      |  128M   | The size of memory size available to the function during execution. Specify a value between 128 MB (default) and 1,536 MB in 128 MB increments. |
+| environment |     Optional      |         | Environment variable of the function                                                                                                            |
+| vpcConfig   |     Optional      |         | VPC configuration of the function                                                                                                               |
+
+- environment param description
+
+| Param     | Description                |
+| --------- | :------------------------- |
+| variables | Environment variable array |
+
+- vpcConfig param description
+
+| Param    | Description      |
+| -------- | :--------------- |
+| subnetId | ID of the VPC    |
+| vpcId    | ID of the subnet |
+
+### apigatewayConf param description
+
+| Param       | Required/Optional | Description                                                                                              |
+| ----------- | :---------------: | :------------------------------------------------------------------------------------------------------- |
+| disable     |     Optional      | Whether disable auto creating api gateway, if you don't want to create it, set to `true`                 |
```

Comment:
When naming boolean properties, it's good to prefix them with `is*`, `has*`. It gives clear indication it's a boolean.

In code `disable` sounds as method, through which we disable given object. So best name here would be `isDisabled`